src/index.js

```diff
@@ -33,7 +33,7 @@ function convertSchemas (context, schemas) {
   }
 }
 
-function jsonSchemasToGraphqlSchema (schemas, withMutations = true) {
```

Comment:
Is it safe to remove the default value? For the sake of keeping the changes small, I'd suggest leaving this line unchanged

Reply:
I believe it's safe, but I agree with you about leaving it unchanged. Thanks!